code/game/mecha/mecha_control_console.dm

```diff
@@ -10,72 +10,85 @@
 
 	l_color = "#CD00CD"
 
-	attack_ai(var/mob/user as mob)
-		src.add_hiddenprint(user)
-		return src.attack_hand(user)
+/obj/machinery/computer/mecha/attack_ai(var/mob/user as mob)
+	src.add_hiddenprint(user)
+	return src.attack_hand(user)
 
-	attack_paw(var/mob/user as mob)
-		return src.attack_hand(user)
-
-	attack_hand(var/mob/user as mob)
-		if(..())
-			return
-		user.set_machine(src)
-		var/dat = "<html><head><title>[src.name]</title><style>h3 {margin: 0px; padding: 0px;}</style></head><body>"
-		if(screen == 0)
-			dat += "<h3>Tracking beacons data</h3>"
-			for(var/obj/item/mecha_parts/mecha_tracking/TR in world)
-				var/answer = TR.get_mecha_info()
-				if(answer)
-					dat += {"<hr>[answer]<br/>
-							  <a href='?src=\ref[src];send_message=\ref[TR]'>Send message</a><br/>
-							  <a href='?src=\ref[src];get_log=\ref[TR]'>Show exosuit log</a> | <a style='color: #f00;' href='?src=\ref[src];shock=\ref[TR]'>(Detonate Beacon)</a><br>"}
-
-		if(screen==1)
-
-			// AUTOFIXED BY fix_string_idiocy.py
-			// C:\Users\Rob\Documents\Projects\vgstation13\code\game\mecha\mecha_control_console.dm:33: dat += "<h3>Log contents</h3>"
-			dat += {"<h3>Log contents</h3>
-				<a href='?src=\ref[src];return=1'>Return</a><hr>
-				[stored_data]"}
-			// END AUTOFIX
+/obj/machinery/computer/mecha/attack_paw(var/mob/user as mob)
+	return src.attack_hand(user)
 
+/obj/machinery/computer/mecha/attack_hand(var/mob/user as mob)
+	if(..())
+		return
+	user.set_machine(src)
+	var/dat = "<html><head><title>[src.name]</title><style>h3 {margin: 0px; padding: 0px;}</style></head><body>"
+	if(screen == 0)
+		dat += "<h3>Tracking beacons data</h3>"
+		for(var/obj/item/mecha_parts/mecha_tracking/TR in world)

```

Comment:
Well you can leave it for now, but tracking beacons should really have a global list they're added to when created and then this would loop through that.